backend/src/main/java/gov/cdc/usds/simplereport/api/heathcheck/BackendAndDatabaseHealthIndicator.java

```diff
@@ -0,0 +1,31 @@
+package gov.cdc.usds.simplereport.api.heathcheck;
+
+import gov.cdc.usds.simplereport.db.repository.FeatureFlagRepository;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.exception.JDBCConnectionException;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+
+@Component("prod-smoke-test")
+@Slf4j
+@RequiredArgsConstructor
+public class BackendAndDatabaseHealthIndicator implements HealthIndicator {
+  @Getter(AccessLevel.NONE)
+  @Setter(AccessLevel.NONE)
+  private final FeatureFlagRepository _repo;
+
+  @Override
+  public Health health() {
+    try {
+      _repo.findAll();
```

Comment:
if there is an okta api call that we can make to verify connectivity (and auth) I think that would be valuable to add here as well.